sdk/src/main/java/com/hashicorp/nomad/apimodel/CsiVolume.java

```diff
@@ -0,0 +1,315 @@
+package com.hashicorp.nomad.apimodel;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.hashicorp.nomad.javasdk.ApiObject;
+import com.hashicorp.nomad.javasdk.NomadJson;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This is a generated JavaBean representing a request or response structure.
+ *
+ * @see <a href="https://www.nomadproject.io/docs/http/index.html">Nomad HTTP API</a> documentation associated with the endpoint you are using.
```

Comment:
It looks like these generated docstrings for the structs all just point to the top-level API?

Reply:
correct